src/components/Permissions/AppList.jsx

```diff
@@ -55,7 +55,7 @@ const AppList = () => {
             .filter(a => !toNotDisplay.includes(a.slug))
             .map(appOrKonnector => {
               return (
-                <div key={appOrKonnector.name}>
+                <div key={appOrKonnector._id}>
```

Comment:
if I have drive1 and drive2, both their names will be drive (same key). So using _id is safer

Reply:
cc @Noemie21 @vdnj